### PR TITLE
Mkirk/style fixups

### PR DESCRIPTION
**Pull request by michaelkirk**
##  side-by-side layout for elevation/route charts

IMO the elevation chart looked pretty bad blown up to full-width on desktop. 
Arguably the route overview looked better at full width (before), but I think it's still legible enough, and overall worth the tradeoff.

**before**
Screen Shot 2021-07-12 at 5 17 16 PM

**after**
Screen Shot 2021-07-12 at 5 16 37 PM

## fix chart style for mdbook

images we scale and crop to fit our layout, but we don't want to do that with charts. This was a bug introduced when porting to mdbook.

**before**
Screen Shot 2021-07-12 at 5 19 38 PM

**after**
Screen Shot 2021-07-12 at 5 20 56 PM

### CODE REVIEW

**Comment by dabreegster** [comment]
All 3 style changes LGTM!

**Comment by michaelkirk** [comment]
I pushed up some additional changes since this is as-of-yet unmerged. If you'd prefer I can open a separate PR.

PTAL!

File: book/src/proposals/broadmoor.md [hunk 1]
```diff
@@ -215,10 +214,9 @@ blockquote {
 </style>
 
 You could be forgiven for not knowing much about the Broadmoor neighborhood in
-Seattle &mdash; that's kind of by design. Using the
-[A / B Street](../software/abstreet.html) travel simulation game, we're going to
-explore how it could become a valuable link for people walking and biking in
-Seattle.
+Seattle &mdash; that's kind of by design. We're going to explore how it could
+become a valuable link for people walking and biking in Seattle, using the
+travel simulation game [A/B Street](../software/abstreet.html).
```
Comment (dabreegster) [inline book/src/proposals/broadmoor.md:219]: Upping the pressure on me to finish this page! This is fine. Pressure's good. :)

File: book/src/proposals/broadmoor.md [hunk 2]
```diff
@@ -388,10 +391,12 @@ how overall trip times and some safety metrics are affected by this proposal.
   <h4 class="chart-header">Cars: Individual Trips</h4>
   <img class="chart" src="../assets/broadmoor/broadmoor-driving-duration-scatter.png" />
 
-For people driving, some trips were a little faster, while others a little
-slower.
+Comparing each drivers trip time before vs. after this change, some trips were a
```
Comment (dabreegster) [inline book/src/proposals/broadmoor.md]: typo, driver's

File: book/src/proposals/broadmoor.md [hunk 3]
```diff
@@ -470,42 +477,74 @@ For example:
...
+including wide streets with fast moving vehicles and many-legged intersections.
 
-South of the Montlake bridge, crossing Highway 520, has complex arterial
-intersections _and_ relatively high bike and foot traffic, meaning many people
-are exposed to these risks.
+Because the Montlake Bridge is one of only a couple plausible connections
+between the University District and central Seattle, it has relatively high bike
+and foot traffic.
+
+The inherent riskiness of these _kinds_ of streets, paired with the high
```
Comment (dabreegster) [inline book/src/proposals/broadmoor.md:502]: I love how you explain this -- there's potential problems inherent to the geometry here, and people are forced to bottleneck through it, so it's an area worthy of consideration.

File: book/src/proposals/broadmoor.md [hunk 4]
```diff
@@ -470,42 +477,74 @@ For example:
...
 
-In shared lanes, a common risk is that faster moving cars pull up too close
-behind a cyclist. This is uncomfortable for both the person on the bike and the
-person driving. Having the option to ride through Broadmoor gives some cyclists
-the option to avoid sharing a lane with cars on busy Madison.
+A 2018 study found that most cyclists who died in single-vehicle crashes were
+struck by the front of the vehicle[^fars_2018].
+
+In shared lanes, a common risk that could lead to this type of collision is when
+a faster moving cars pull up too close behind a cyclist. This is uncomfortable
```
Comment (dabreegster) [inline book/src/proposals/broadmoor.md]: Grammar: cars pull -> car pulls

**Review by dabreegster — APPROVED** [review]
Minus the few typos, LGTM! Spectacular!

**Comment by michaelkirk** [comment]
Fixed the typos, thanks!

**Comment by dabreegster** [comment]
http://abstreet.s3-website.us-east-2.amazonaws.com/0.2.51/abstreet.html?--dev&system/us/seattle/scenarios/arboretum/weekday.bin&--edits=broadmoor%20access  is now live

**Pull request merged**